test/cmdlineTests/function_debug_info/output

```diff
@@ -35,20 +35,26 @@
         },
         "abi_decode_tuple_t_array$_t_uint256_$dyn_calldata_ptr":
         {
-          "entryPoint": 178,
+          "entryPoint": 164,
           "parameterSlots": 2,
           "returnSlots": 2
         },
         "abi_decode_tuple_t_uint256":
         {
-          "entryPoint": 295,
+          "entryPoint": 281,
           "parameterSlots": 2,
           "returnSlots": 1
         },
         "abi_encode_tuple_t_uint256__to_t_uint256__fromStack_reversed":
         {
           "parameterSlots": 2,
```

Comment:
Unrelated question, why is this entry missing `entryPoint`, @chriseth?

Reply:
Maybe because it is inlined or deemed equivalent to another function?

Reply:
Shouldn't it have the an `entryPoint` with the matching offset then, so it is easy to find equivalent functions?

Reply:
I think it would require the optimizer to be aware of this debug data and track it all the way through all the steps. The optimizer would have to modify it when it inlines a function or replaces it with an equivalent. Currently the logic is a lot simpler than that (see #8868). We add named tags for all functions at the beginning and at the end, when we generate the bytecode and iterate over assembly items, every named tag gets a bytecode offset assigned. We do not track what happens to the tag in between. If it does not appear in the final bytecode, we just get no offset for the entry point.

The entry point is present when you compile with `--optimize` which confirms that this is the optimizer doing that:
```json
        "abi_encode_tuple_t_uint256__to_t_uint256__fromStack_reversed":
        {
          "entryPoint": 498,
          "parameterSlots": 2,
          "returnSlots": 1
        },
```